### PR TITLE
theme native scrollbars + scrollbar fixes

### PR DESCRIPTION
**Pull request by titoBouzout**

### CODE REVIEW

**Review by LadyBluenotes — APPROVED** [review]

**Comment by LadyBluenotes** [comment]
@atilafassina / @danieljcafonso  can you check this and merge if you're good with it?

File: src/styles.css [hunk 1]
```diff
@@ -16,8 +17,34 @@
...
+	/** Scroll bar */
+
+	/**
+	 * `html`
+	 * Applies colors to the main *native* browser scrollbar.
+	 *
+	 * `html.firefox .custom-scrollbar`
+	 * On firefox all scrollbars are native. This styles them with our custom colors.
+	*/
+	html,
```
Comment (JasonHassold) [inline src/styles.css:30]: @titoBouzout we should leave the main browser window scrollbars default. Custom styling for scrollbars should only be only the in-page content 
```suggestion
```

File: src/styles.css [hunk 2]
```diff
@@ -16,8 +17,34 @@
...
+	html.firefox .custom-scrollbar {
+		scrollbar-color: #778292 #e7ecf2;
+		scrollbar-width: thin;
+	}
+	html.dark,
```
Comment (JasonHassold) [inline src/styles.css:35]: @titoBouzout same as above
```suggestion
```

**Review by JasonHassold — CHANGES_REQUESTED** [review]
@titoBouzout small change request

File: src/styles.css (hunk 1 above)
Comment (titoBouzout) [reply src/styles.css:30]: Is not a custom scrollbar, its a native scrollbar styled

Comment (titoBouzout) [reply src/styles.css:30]: It also themes scrollbars on firefox, which doesnt allow custom scrollbars

Comment (JasonHassold) [reply src/styles.css:30]: Sorry I more just mean default vs non-default. I don't think this is a big issue or anything so feel free to keep it. It was just push back I got on the custom scrollbars before and the compromise was to only alter the in-page scrollbars and keep the main page ones the browser default

Comment (titoBouzout) [reply src/styles.css:30]: No worries, the main issue is replacing the native scrollbars, as long as the main scrollbar keeps being native is a non-issue, and I believe the styled scrollbar looks better overall (while still being native)

**Pull request merged**